cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -543,10 +539,10 @@
             {# Currently this applies to 0.4+ #}
             <tr>
                 <td>
-                    {% blocktrans %}Death date is incorrect.{%endblocktrans%}
+                    {% blocktrans %}<code>deathDate</code> is incorrect. Check that dates are correctly generated and well formatted.{% endblocktrans %}
```

Comment:
could we give a bit more explanation here? "Check the date is not before the birthDate or in the future"?

Reply:
Will do.